### PR TITLE
Release Notes 2.11.1

### PR DESCRIPTION
**Pull request by pjfitzgibbons**
Signed-off-by: Peter Fitzgibbons <peter.fitzgibbons@gmail.com>

### Description
Release Notes 2.11.1

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-dashboards-reporting.release-notes-2.11.1.0.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Version 2.11.1 Release Notes
+
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.11.0
+
+### Features
+Use core navigation instead of hard coding URLs ([#229](https://github.com/opensearch-project/dashboards-reporting/pull/229)) ([#231](https://github.com/opensearch-project/dashboards-reporting/pull/231))
```
Comment (smortex) [inline release-notes/opensearch-dashboards-reporting.release-notes-2.11.1.0.md:6]: Feature? In a PATCH release?

https://semver.org/#spec-item-6
> Patch version Z (x.y.Z | x > 0) MUST be incremented if only backward compatible bug fixes are introduced. A bug fix is defined as an internal change that fixes incorrect behavior.

Have not all non-bugfix backports been revert?

File: release-notes/opensearch-dashboards-reporting.release-notes-2.11.1.0.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## Version 2.11.1 Release Notes
+
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.11.0
+
+### Features
```
Comment (anirudha) [inline release-notes/opensearch-dashboards-reporting.release-notes-2.11.1.0.md:5]: @pjfitzgibbons can you cross check/ this looks like a maintainence or bug fix / not a feature

**Review by anirudha — CHANGES_REQUESTED** [review]
review features

File: release-notes/opensearch-dashboards-reporting.release-notes-2.11.1.0.md (hunk 2 above)
Comment (pjfitzgibbons) [reply release-notes/opensearch-dashboards-reporting.release-notes-2.11.1.0.md:5]: Yes, correct, #229 fixes #227.  Not a feature.

**Review by anirudha — APPROVED** [review]

**Pull request merged**